typescript/infra/package.json

```diff
@@ -14,9 +14,9 @@
     "@gnosis.pm/safe-core-sdk": "^2.3.2",
     "@gnosis.pm/safe-ethers-lib": "^1.4.0",
     "@gnosis.pm/safe-service-client": "^1.2.0",
-    "@hyperlane-xyz/helloworld": "1.3.5",
-    "@hyperlane-xyz/sdk": "1.3.5",
-    "@hyperlane-xyz/utils": "1.3.5",
+    "@hyperlane-xyz/helloworld": "1.3.6",
```

Comment:
Update the infra version to 1.3.6 as well. It doesn't happen automatically cause the package is private

Reply:
ah thanks